frontend/src/__tests__/cypress/cypress/pages/components/TableToolbar.ts

```diff
@@ -0,0 +1,19 @@
+class TableToolbar {
+  private findToggleButton(id: string) {
+    return cy.findByTestId(id).find('.pf-v5-c-dropdown__toggle').click();
+  }
+
+  findFilterMenuOption(id: string, name: string) {
+    return this.findToggleButton(id).parents().findByRole('menuitem', { name }).click();
+  }
+
+  findSearchInput() {
+    return cy.findByRole('textbox', { name: 'Search input' });
+  }
```

Comment:
Can we use test ids here.

Reply:
I tried using test id, but it doesn't seems to work for `searchInput`. You want me to use `findByLabelText` here instead of `findByRole`